### PR TITLE
[12.0] [MIG] hr_employee_phone_extension

### PR DESCRIPTION
**Pull request by anandkansagra**
Issue [Reference](https://github.com/OCA/hr/issues/479)

### CODE REVIEW

**Review by nikul-serpentcs — APPROVED** [review]
If https://github.com/OCA/hr/pull/518#issuecomment-442452417 is fine then Code review LGTM

File: hr_employee_phone_extension/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# copyright 2013 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (anandkansagra) [inline hr_employee_phone_extension/__init__.py]: @nikul-serpentcs ,
I think we need to remove the license from `__init__.py` file right?
Well, I have removed it and squash the commit.

File: hr_employee_phone_extension/views/hr_employee_view.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version = "1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Inherited employee form view -->
+    <record id="view_employee_form_inherit_extension" model="ir.ui.view">
```
Comment (tbaden) [inline hr_employee_phone_extension/views/hr_employee_view.xml]: aesthetic change (optional):
use original id (view_employee_form)

> prefer naming the inherited view after the original over using a name which follows the guidelines. This eases looking up the original view and other inheritance if they all have the same name.

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
I did a code review:
please use the new fragment [readme](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration)

**Comment by anandkansagra** [comment]
@tbaden ,
I have made the changes. Could you please look it out?
Thanks

**Review by tbaden — APPROVED** [review]
LGTM

**Pull request merged**